src/DbUtils.php

```diff
@@ -514,11 +514,15 @@ public function countElementsInTable($table, $condition = [])
        }*/
 
         if (!is_array($condition)) {
+            Toolbox::Deprecated('Condition must be an array!');
             if (empty($condition)) {
                 $condition = [];
             }
         }
-        $condition['COUNT'] = 'cpt';
+
+        if (is_array($condition)) {
+            $condition['COUNT'] = 'cpt';
```

Comment:
I guess `return ($row ? (int)$row['cpt'] : 0);` will not able to find `cpt` when condition is a string. Not sure it will work.